AGENTS.md

```
@@ -7,6 +7,13 @@
 zsh -lc "build-rmcs --packages-up-to rmcs_auto_aim_v2"
 ```
 
+如果位于 `test` 或者 `tool/cxx` 下，则使用下面指令独立构建
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)